web/react/utils/client.jsx

```diff
@@ -279,24 +279,33 @@ module.exports.getAudits = function(userId, success, error) {
     });
 };
 
-module.exports.getMe = function(success, error) {
+module.exports.getMeSynchronous = function(success, error) {
+    var currentUser = null;
 
     $.ajax({
+        async: false,
```

Comment:
It is not being deprecated. The use of ```aysnc: false``` with promises (.done(), .fail(), etc.) is being deprecated but we don't use promises here so it's fine. This is why I'm adding the code back in.

http://api.jquery.com/jquery.ajax/